spec/features/providers/details_spec.rb

```diff
@@ -44,19 +56,37 @@
     end
   end
 
-  def test_details_page(expected_status)
-    stub_api_v2_request(
-      "/recruitment_cycles/#{provider.recruitment_cycle.year}",
-      provider.recruitment_cycle.to_jsonapi
-    )
-    stub_api_v2_request(
-      "/recruitment_cycles/#{provider.recruitment_cycle.year}" \
-        "/providers/#{provider.provider_code}",
-      provider.to_jsonapi
-    )
+  context "Current recruitment cycle" do
+    let(:provider) do
+      build :provider,
+            provider_code: 'AO',
+            content_status: 'draft',
+            recruitment_cycle: build(:recruitment_cycle, year: Settings.current_cycle)
```

Comment:
Doesn't the factory take care of all this for you? I wouldn't bother if you don't need to, adds noise and there's no dependency (that I can see) that the code is `A0`, is there?